_layouts/training.html

```diff
@@ -19,11 +19,10 @@ <h2 class="service-header">Training</h2>
                 <div id="contact" class="headline headline-md margin-bottom-30">
                     <h2>Talk to us about your training needs</h2>
                 </div>
-
-                <form id="training-contact-form" class="sky-form" action="https://codurance.com/api/emailer">
+                <form id="training-contact-form" class="sky-form" action="#" onsubmit="event.preventDefault(); submitTrainingContactForm();">
                     <input type="hidden" name="_next" value="{{site.baseurl}}/aboutus/thanks"/>
-                    <input type="hidden" name="_subject" value="Codurance Training"/>
-                    <input type="hidden" name="to" value="hello@codurance.com"/>
+                    <input type="hidden" name="subject" value="Codurance Training"/>
+                    <input type="hidden" name="to" value="join-us@codurance.com"/>
```

Comment:
DIscussed with @davidgrahamhall, he wanted it to be join-us for the time being, as for us to not re-do integration with Trello. In the meantime he's looking into having more wholesome solution to the email pipeline